docs/developer-guide/serialization.rst

```diff
@@ -373,6 +373,34 @@ Field Name Case Sensitivity
 The format is case-insensitive when matching a KSQL field name with an Avro record's field name.
 The first case-insensitive match is used.
 
+Decimal Serialization
+---------------------
+
+KSQL accepts Decimals that are serialized either as numbers, or the text representation of the
+base 10 equivalent. For example, KSQL can read data from both formats below:
+
+.. code:: json
+
+        {
+          "value": 1.12345678912345,
+          "value": "1.12345678912345"
+        }
+
+
+Decimals with specified precision and scale are serialized as JSON floating point numbers. For
+example:
+
+.. code:: json
+
+        {
+          "value": 1.12345678912345
+        }
+
+Any downstream application that reads JSON output from a KSQL application that produces decimals
```

Comment:
Personally, I think this paragraph is superfluous. KSQL is output valid / correct JSON.  Calling out that any downstream app should handle this correctly is just a bit: well dur!  It's also calling out Java specific stuff, like Jackson, where the downstream processing could be anything, (especially as KSQL is for none-java users).